.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,42 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: Chromatic Publish and Testing
+
+# Event for the workflow
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
+    paths:
+      # Only run on file changes in any of these paths
+      - "src/components/**/*"
+      - "src/pages/**/*"
+      - "src/pages-conditional/**/*"
+      - "src/templates/**/*"
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      # 👇 Version 2 of the action
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+      - name: Install deps
+        # 👇 Install dependencies with the same package manager used in the project (replace it as needed), e.g. yarn, npm, pnpm
+        run: yarn
+      - name: Publish to Chromatic
+        # 👇 Adds Chromatic as a step in the workflow
+        uses: chromaui/action@v1
+        # Options required for Chromatic's GitHub Action
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          # 👇 Only fail if Storybook contains stories that error
+          exitZeroOnChanges: true
```

Comment:
Along with these options, `onlyChanged` can be considered in the future. This enables Chromatic's TurboSnap to speed up the testing process by only updating snapshots of stories related to the files that have changed.

Can be an alternative or an addition to [lines 13 to 18](https://github.com/ethereum/ethereum-org-website/pull/9234/files#r1080664656)

See [TurboSnap](https://www.chromatic.com/docs/turbosnap)

Reply:
Cool. Yeah, it seems they don't let you enable it immediately when you start a project anyway.

Agree with you. To start with, I think we're good with the current setup.